pygeodiff/tests/test_skip_tables.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+    :copyright: (c) 2022 Alexander Bruy
+    :license: MIT, see LICENSE for more details.
+"""
+
+from .testutils import *
+import os
+import shutil
+import pygeodiff
+
+class UnitTestsPythonSingleCommit(GeoDiffTests):
+    def test_skip_create(self):
+        base = testdir() + "/" + "skip_tables" + "/" + "base.gpkg"
```

Comment:
is the base.gpkg table the same as in testdata/base.gpkg ? If so, I would suggest not to create another copy and just use the one from testdata/base.gpkg

Reply:
It is different, this file has several layers, so we can test whether excluding tables works.